CONTRIBUTING.md

```diff
@@ -25,21 +25,45 @@ Overall, vignette submission will follow the guidelines in [GitHub flow](https:/
 Since this process requires git, if you are having trouble installing git or accessing your command line, please take a look at our detailed tutorial on using git through R: [Contributing with git2r](CONTRIBUTING_WITH_GIT2R.html).
 
  
+## Setup for new contributors
+
+If you are a new contributor, you will need to make sure you have git set up and a fork of this repository. Please follow these instructions carefully. If you have already set up your repository, go to the next section. 
+
+1. Set up a GitHub account if you don't already have one, see [here](https://guides.github.com/activities/hello-world/) for a quick tutorial on using GitHub. 
+2. Download an appropriate Git client for your computer, see [here](https://help.github.com/articles/set-up-git/), and set it up.
+3. Fork the [source repository](http://github.com/NESCent/popgenInfo) on GitHub using the `Fork` button (See [GitHub's tutorial](https://help.github.com/articles/fork-a-repo/)). 
+4. On the command line, clone the repository to your machine using    
+`git clone https://github.com/YourAccount/popgenInfo.git`    
+Note: the https protocol is easier to follow, but it's possible to use ssh as well ([details here](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork)). 
+5. Add the NESCent repository to upstream with     
+`git remote add NESCent https://github.com/NESCent/popgenInfo.git`    
```

Comment:
You have a good point. I can change this easily, but it will also mean that I will also need to change the git2r vignette and inform our collaborators to rename their remote `upstream`.

I will make further commits.